tests/utils/shippable/shippable.sh

```diff
@@ -77,28 +94,21 @@ pip install setuptools==44.1.0
 
 pip install https://github.com/ansible/ansible/archive/"${A_REV:-devel}".tar.gz --disable-pip-version-check
 
-#ansible-galaxy collection install community.general
-mkdir -p "${HOME}/.ansible/collections/ansible_collections/community"
-mkdir -p "${HOME}/.ansible/collections/ansible_collections/google"
-mkdir -p "${HOME}/.ansible/collections/ansible_collections/openstack"
-cwd=$(pwd)
-cd "${HOME}/.ansible/collections/ansible_collections/"
-git clone https://github.com/ansible-collections/community.general community/general
-git clone https://github.com/ansible-collections/community.aws community/aws
-# community.general requires a lot of things we need to manual pull in
-# once community.general is published this will be handled by galaxy cli
-git clone https://github.com/ansible-collections/ansible_collections_google google/cloud
-git clone https://opendev.org/openstack/ansible-collections-openstack openstack/cloud
-ansible-galaxy collection install ansible.netcommon
-cd "${cwd}"
-
 export ANSIBLE_COLLECTIONS_PATHS="${HOME}/.ansible/"
 SHIPPABLE_RESULT_DIR="$(pwd)/shippable"
 TEST_DIR="${HOME}/.ansible/collections/ansible_collections/amazon/aws/"
 mkdir -p "${TEST_DIR}"
 cp -aT "${SHIPPABLE_BUILD_DIR}" "${TEST_DIR}"
 cd "${TEST_DIR}"
 
+# START: HACK install dependencies
+retry ansible-galaxy collection install ansible.netcommon
+#retry ansible-galaxy collection install ansible.windows
+retry git clone https://github.com/ansible-collections/ansible.windows.git "${ANSIBLE_COLLECTIONS_PATHS}/ansible_collections/ansible/windows"
+retry ansible-galaxy collection install community.aws
```

Comment:
One thought about this (thinking ahead to when we'd update community.aws's dependency installation) - assuming we install amazon.aws from galaxy we wouldn't be able to test updates to amazon.aws breaking things in community.aws until after it lands...

Probably better to land community.aws and amazon.aws directly from GitHub